src/CoreAnimation/CAFrameRateRange.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+using Foundation; 
+using ObjCRuntime;
+
+#nullable enable
+
+namespace CoreAnimation {
+
+#if !NET
+	[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+#else
+	[SupportedOSPlatform ("ios15.0"), SupportedOSPlatform ("tvos15.0"), SupportedOSPlatform ("macos12.0"), SupportedOSPlatform ("maccatalyst15.0")]
+#endif
+	[StructLayout (LayoutKind.Sequential)]
+	public struct CAFrameRateRange
+	{
+		public float Minimum;
+
+		public float Maximum;
+
+		public float Preferred;
+
+		[DllImport(Constants.QuartzLibrary, EntryPoint = "CAFrameRateRangeIsEqualToRange")]
+		[return: MarshalAs (UnmanagedType.I1)]
+		static extern bool IsEqualTo (CAFrameRateRange range, CAFrameRateRange other);
+
+		[DllImport(Constants.QuartzLibrary, EntryPoint = "CAFrameRateRangeMake")]
+		public static extern CAFrameRateRange Create (float minimum, float maximum, float preferred);
+
+		public bool IsEqualTo (CAFrameRateRange other)
+			=> IsEqualTo  (this, other);
+
+#if !COREBUILD
+		[Field ("CAFrameRateRangeDefault", "CoreAnimation")]
+ 		public static CAFrameRateRange Default => (CAFrameRateRange) Marshal.PtrToStructure (Dlfcn.GetIndirect (Libraries.CoreAnimation.Handle, "CAFrameRateRangeDefault"), typeof (CAFrameRateRange))!;
```

Comment:
The generic version of PtrToStructure doesn't work?

```suggestion
 		public static CAFrameRateRange Default => Marshal.PtrToStructure<CAFrameRateRange> (Dlfcn.GetIndirect (Libraries.CoreAnimation.Handle, "CAFrameRateRangeDefault"))!;
```